src/ride_type.cpp

```diff
@@ -201,17 +201,17 @@ void SetRideRatingStringParam(const uint32 rating)
 	std::string str;
 
 	if (rating < 100) {
-		str = _language.GetText(GUI_RIDE_MANAGER_RATING_VERY_LOW);
+		str = _language.GetSgText(GUI_RIDE_MANAGER_RATING_VERY_LOW);
 	} else if (rating < 250) {
-		str = _language.GetText(GUI_RIDE_MANAGER_RATING_LOW);
+		str = _language.GetSgText(GUI_RIDE_MANAGER_RATING_LOW);
 	} else if (rating < 500) {
-		str = _language.GetText(GUI_RIDE_MANAGER_RATING_MEDIUM);
+		str = _language.GetSgText(GUI_RIDE_MANAGER_RATING_MEDIUM);
 	} else if (rating < 750) {
-		str = _language.GetText(GUI_RIDE_MANAGER_RATING_HIGH);
+		str = _language.GetSgText(GUI_RIDE_MANAGER_RATING_HIGH);
 	} else if (rating < 1050) {
-		str = _language.GetText(GUI_RIDE_MANAGER_RATING_VERY_HIGH);
+		str = _language.GetSgText(GUI_RIDE_MANAGER_RATING_VERY_HIGH);
```

Comment:
Random entry, but why is singular form so special? Wouldn't 0 parameters do the job?